### ISSUE TITLE
Multiple websites at once?

### ISSUE THREAD

**Issue by heyfletch** [BuilderIO/gpt-crawler#135]
Is there a config to crawl multiple websites? For example, example.com and mysite.com?

**Comment by julian-passebecq** [comment]
yes i did a custom version, i give him a list of url, it will scrawl all of them and all their /*

**Comment by julian-passebecq** [comment]
but its not only on custom, i changed different part

**Comment by heyfletch** [comment]
> yes i did a custom version, i give him a list of url, it will scrawl all of them and all their /*

That's great! Any chance you can share an example?

**Comment by julian-passebecq** [comment]
Yes but how to do ? I publish on my GitHub ? Or I have to do a pr ?

**Comment by heyfletch** [comment]
Maybe start by pasting your example into a comment here.

**Comment by julian-passebecq** [comment]
https://github.com/julian-passebecq/gpt-crawl-multiurl
i put the instructions in readme

**Comment by heyfletch** [comment]
I don't understand your comment, "don't use config ts for the url to scrawl, only modify json name in config.ts put the url to crawl in tocrawl.csv". Could you please explain that more? Meaning, which file do I modify, and how do I modify it to crawl "tocrawl.csv"

**Comment by julian-passebecq** [comment]
before, you had to put the url to scrawl in config.ts. now, you only put the name of the json in config.ts, the url written there doesn't matter. yo modify tocrawl, you can do it manually, just respect the 3 columns format i used. me i use allcrawl.csv, i put all my url, then i use crawlee.py, to only input in tocrawl.csv 1 topic of several url to scrawl

**Comment by heyfletch** [comment]
Ok, thanks for the clarification!!

**Comment by julian-passebecq** [comment]
so, did you try my multi url version ? what do u think ?

**Comment by heyfletch** [comment]
Haven't tried it yet, but if I do, I'll let you know. :)
